src/Shell.php

```diff
@@ -1134,6 +1139,22 @@ public function writeReturnValue($ret, bool $rawOutput = false)
         }
     }
 
+    /**
+     * Checks if the "gray" color exists on the output.
+     *
+     * @return bool
+     */
+    private function grayExists(): bool
+    {
+        try {
+            $this->output->write('<fg=gray></>');
```

Comment:
probably don't need to try this every time we write something to output :) 

if we're using named styles, maybe we check once when defining the styles (`<whisper>`?) and make it a no-op if grey doesn't work?